extensions/wikia/CommunityPage/models/CommunityPageSpecialTopAdminsFormatter.class.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+class CommunityPageSpecialTopAdminsFormatter {
+	const TOP_ADMINS_LIMIT = 10,
+		TOP_ADMINS_MODULE_LIMIT = 3;
+
+	/**
+	 * Returns array with fields to supply topAdmins.mustache template
+	 * If there are more than three admins provided returns first two and the count of remaining ones
+	 * @param array $topAdminsDetails
+	 * @return array
+	 */
+	public static function prepareData( array $topAdminsDetails ) {
+		$topAdminsCount = count( $topAdminsDetails );
+		$remainingAdminCount = self::prepareRemainingCount( $topAdminsCount );
+		return [
+			'admins' => self::prepareAdminsToShow( $topAdminsCount, $remainingAdminCount, $topAdminsDetails ),
+			'otherAdminCount' => $remainingAdminCount,
+			'haveOtherAdmins' => $remainingAdminCount > 0,
+			'adminCount' => $topAdminsCount,
+		];
+	}
+
+	private function prepareRemainingCount( $topAdminsCount ) {
+		return $topAdminsCount > self::TOP_ADMINS_MODULE_LIMIT
+			? $topAdminsCount - self::TOP_ADMINS_MODULE_LIMIT + 1
```

Comment:
Our [conventions](https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP#Ternary_operator) for ternary operators in PHP are that they are fine but to prefer `if` if it's a multi-line expression for readability (and as a side note, definitely don't want to nest ternary operators). An `if` block to be slightly more readable here, but it's not a big problem either way for a relatively simple one like this.